### ISSUE TITLE
Add mpw2 tapeout gds and info here

### ISSUE THREAD

**Issue by msaligane** [idea-fasoc/openfasoc-tapeouts#4]

**Comment by msaligane** [comment]
@lulu9312 and @ayushman22git
Can you please add the gds and documentation from MPW2 here.

**Comment by lulu9312** [comment]
@msaligane Yes. I have added gds files and some basic information regarding the ldo components in this PR #10 . @ayushman22git Can you share some basic details about the mpw-2 test board so that we can add those as well.
